src/objects/token/tests/test_admin.py

```diff
@@ -0,0 +1,84 @@
+from django.test import TestCase, tag
+from django.urls import reverse
+
+from maykin_2fa.test import disable_admin_mfa
+from vcr.unittest import VCRMixin
+from zgw_consumers.constants import AuthTypes
+from zgw_consumers.test.factories import ServiceFactory
+
+from objects.accounts.tests.factories import UserFactory
+from objects.core.tests.factories import ObjectTypeFactory
+
+
+@disable_admin_mfa()
+class PermissionAdminTests(VCRMixin, TestCase):
+
+    object_types_api = "http://127.0.0.1:8008/api/{version}/"
+
+    def setUp(self):
+        super().setUp()
+
+        self.user = UserFactory(superuser=True)
+        self.client.force_login(self.user)
+        self.url = reverse("admin:token_permission_add")
+
+    def _get_vcr(self, **kwargs):
+        vcr = super()._get_vcr(**kwargs)
+        vcr.filter_headers = ["authorization"]
+        return vcr
+
+    @tag("#449")
+    def test_with_object_types_api_v1(self):
+        """
+        Regression test for #449.
+        Test if Permission admin can still handle objecttypes API V1
+        """
+
+        v1_service = ServiceFactory(
+            api_root=self.object_types_api.format(version="v1"),
+            auth_type=AuthTypes.api_key,
+            header_key="Authorization",
+            header_value="Token 5cebbb33ffa725b6ed5e9e98300061218ba98d71",
+        )
+        object_type = ObjectTypeFactory(
+            service=v1_service, uuid="71a2452a-66c3-4030-b5ec-a06035102e9e"
+        )
+
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+
+        form = response.context["adminform"].form
+        choices = list(form.fields["object_type"].choices)
+
+        self.assertEqual(
+            choices[1][0].value,
+            object_type.id,
+        )
+
+    @tag("#449")
+    def test_with_object_types_api_v2(self):
+        """
+        Regression test for #449.
+        Test if Permission admin can handle objecttypes API V2 which added pagination
+        """
+        v2_service = ServiceFactory(
+            api_root=self.object_types_api.format(version="v2"),
+            auth_type=AuthTypes.api_key,
+            header_key="Authorization",
+            header_value="Token 5cebbb33ffa725b6ed5e9e98300061218ba98d71",
+        )
+        object_type = ObjectTypeFactory(
+            service=v2_service, uuid="71a2452a-66c3-4030-b5ec-a06035102e9e"
+        )
+
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+
+        form = response.context["adminform"].form
+        choices = list(form.fields["object_type"].choices)
+
+        self.assertEqual(
+            choices[1][0].value,
+            object_type.id,
```

Comment:
bit of a nitpick, but is it possible to check the label here as well? Currently it will probably just check if the `value` is `1`, which doesn't really say a lot :grimacing: